strapi/strapi
# [Audit Logs] Add pagination to Audit logs page

### What does it do?

We add the pagination component to the audit logs page so you can navigate between pages and set a pageSize number

### How to test it?

Run a Strapi application with EE enabled and go to the Audit Logs page, if you have more than 10 events you should see multiple pages, you should move between pages without problem and you should edit the pageSize without problem
